src/components/engagement/engagement.service.ts

```diff
@@ -593,9 +594,14 @@ export class EngagementService {
 
   /**
    * [BUSINESS RULE] Only Projects with a Status of 'In Development' can have Engagements created or deleted.
+   * [BUSINESS RULE] Only Projects with a Status of 'Active' and part of a changeset can have Engagements created or deleted.
    */
-  protected async verifyProjectStatus(projectId: ID, session: Session) {
-    if (session.roles.includes('global:Administrator')) {
+  protected async verifyProjectStatus(
+    projectId: ID,
+    session: Session,
+    changeset?: ID
+  ) {
+    if (changeset || session.roles.includes('global:Administrator')) {
```

Comment:
So this is needs to be taken out?

Reply:
That can stay. The query for the change request list is what should be taken out.